src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.Core/HostedServices/Services/PartySyncService.cs

```diff
@@ -225,6 +225,7 @@ private Entity CreateEntity(Party party, Action<Entity> configureEntity)
             UserId = party?.User.Value?.UserId.HasValue == true ? Convert.ToInt32(party.User.Value.UserId.Value) : null,
             Username = party?.User.Value?.Username.HasValue == true ? party.User.Value.Username.ToString() : null,
             IsDeleted = party?.IsDeleted.HasValue == true ? party.IsDeleted.Value : false,
+            Audit_ValidFrom = party.ModifiedAt.Value,
```

Comment:
Litt usikker på om jeg er enig i at vi skal bruke ModifiedAt som Audit_ValidFrom.

Audit er for å spore hvem og når en endring blir gjort i våre lokale data. Ikke for når den ble endret på eksternt.